assembly/etc/che-plugin.yaml

```diff
@@ -8,7 +8,7 @@ endpoints:
       discoverable: false
 containers:
  - name: che-machine-exec
-   image: eclipse/che-machine-exec
+   image: aandrienko/che-machine-exec:latest
```

Comment:
hello, why is aandrienko ?

Reply:
Yes, it's was not needed changes. Thanks.